Packages/vcs/vcs/utils.py

```diff
@@ -312,42 +312,6 @@ def _scriptrun(script, canvas=None):
                 vcs.process_src_element(s)
                 processing = False
     f.close()
-    # Ok now we need to double check the isolines
```

Comment:
@danlipsa without taking a deep look I can tell you what I "remember" 
isoline graphic methods that had styles and colors used to be stored in the old `.scr` files as separte `line` objects.
I think this bit of code as something to do with this.